### PR TITLE
fix: BottomNavigation, AppHeader - center items and add mobile menu

### PR DESCRIPTION
**Pull request by AdamPawlinski**
## Description
Added mobile menu in AppHeader to be opened on click in Bottom Navigation. 
BottomNavigation items centered. 

## Related Issue
#184 

## Screenshots (if appropriate):
vendure_mobile_menu

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/components/BottomNavigation.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<template>
+<!-- TODO: create logic with isActive prop for BottomNavigationItems -->
```
Comment (Baroshem) [inline packages/theme/components/BottomNavigation.vue:2]: Could you please create github issues for implementing these TODOs? :)

File: packages/theme/components/BottomNavigation.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  <SfBottomNavigation class="navigation-bottom smartphone-only">
+      <SfBottomNavigationItem
+        :class="$route.path == '/' ? 'sf-bottom-navigation__item--active' : ''"
+        icon="home" size="20px" label="Home"
```
Comment (Baroshem) [inline packages/theme/components/BottomNavigation.vue:6]: I think here something is broken with the linter as some attributes are in the same line while other are moved to the next on

File: packages/theme/components/BottomNavigation.vue [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        @click="handleHomeClick"
+      />
+    <SfBottomNavigationItem icon="menu" size="20px" label="Menu" @click="toggleMobileMenu"/>
```
Comment (Baroshem) [inline packages/theme/components/BottomNavigation.vue:9]: Could you please add i18n to the labels here?

File: packages/theme/components/AppHeader.vue [hunk 4]
```diff
@@ -11,14 +11,33 @@
...
+          <SfHeaderNavigationItem
+            v-for="category in headerNavigation"
+            :key="category.name"
+            class="nav-item"
+            v-e2e="`app-header-${category.name}`"
+            :label="category.name"
+            :link="localePath(`/c/${category.link}`)"
+          />
+        </div>
+        <SfModal v-else :visible="isMobileMenuOpen" title="Menu" @close="toggleMobileMenu">
```
Comment (Baroshem) [inline packages/theme/components/AppHeader.vue:24]: Here also i18n

**Review by Baroshem — COMMENTED** [review]
Few small recommendations to include. Overall, very good!

File: packages/theme/components/BottomNavigation.vue (hunk 2 above)
Comment (AdamPawlinski) [reply packages/theme/components/BottomNavigation.vue]: Ok done.

File: packages/theme/components/BottomNavigation.vue (hunk 3 above)
Comment (AdamPawlinski) [reply packages/theme/components/BottomNavigation.vue]: done

File: packages/theme/components/AppHeader.vue (hunk 4 above)
Comment (AdamPawlinski) [reply packages/theme/components/AppHeader.vue]: done

File: packages/theme/components/BottomNavigation.vue (hunk 1 above)
Comment (AdamPawlinski) [reply packages/theme/components/BottomNavigation.vue:2]: done

**Review by Baroshem — APPROVED** [review]

**Comment by ashishkpaul** [comment]
Menu navigation working through git clone repo but not working with cli installation
as well as user account page (http://example.com/my-account) hang when repo clone through git but not hanging with cli installation.
please also read https://github.com/vuestorefront/vue-storefront/issues/6820#issue-1434223917

**Pull request merged**